client/state/themes/actions/activate-theme.js

```diff
@@ -31,9 +31,11 @@ export function activateTheme(
 			siteId,
 		} );
 
-		return wpcom
-			.undocumented()
-			.activateTheme( themeId, siteId, keepCurrentHomepage )
+		return wpcom.req
+			.post( `/sites/${ siteId }/themes/mine`, {
+				theme: themeId,
+				...( dontChangeHomepage && { dont_change_homepage: true } ),
```

Comment:
Another case where the PHP impl checks only the param presence and not its value 😢

Reply:
Yeah, definitely not ideal.